### ISSUE TITLE
Use bower for jquery and bootstrap

### ISSUE THREAD

**Issue by michael-conway** [DICE-UNC/irods-cloud-browser#2]
switch to bower for jquery and bootstrap dependencies

**Comment by PaulVanSchayck** [comment]
@michael-conway We are using the cloud browser in one of our projects. Thank you for it, we love the use of angular in it. We're also modifying it to fit our project needs a bit more.

However, this issue, and in particular the fact that the content of node_modules and bower_components has been included in the git repository makes our release/development process harder. We are unable to easily add new npm dependencies or to update existing without making changing the entire repository. That in turn makes it more difficult to merge your upstream changes back into our fork.

Would you be able to fix this, or accept a new PR to fix this?

**Comment by michael-conway** [comment]
yep, this is a serious problem.  I'd love a pull request.  It was not
configured properly but we had to roll with it.

**Comment by PaulVanSchayck** [comment]
OK, I've got a version of gulp.js, package.json and bower.json that is able to build a working cloud browser into the `dist/` directory. Needs a few more cleanups..

What is your idea? Do you want to keep a working version in `app/` as well, or do you want to use `gulp dist` also during development?

**Comment by michael-conway** [comment]
I want to develop out of dist with automation to test/build when files 
are changed. This was the original setup when it was 'idrop-web3'.

Ideally we can eventually automate it to be able to package the static 
resources into the .war itself, this was also originally in 
'idrop-web3'.  That part sort of got broken when this transitioned into 
the consortium devs.

**Comment by PaulVanSchayck** [comment]
That makes sense. So you would like to make use of the watch() feature of gulp?

I've updated and changed my PR #153 to fix loading dependencies, and the building through gulp. Is this what you are looking for?

**Comment by PaulVanSchayck** [comment]
Hi Mike,

Did you have a chance to look at this PR #153? 

Thanks,

Paul

**Comment by michael-conway** [comment]
Yes, we've already done some initial work on it, and it will be 
integrated for this coming release.  I had actually been trying to 
untangle some of this already!
